app/models/note.rb

```diff
@@ -4,5 +4,10 @@ class Note < ApplicationRecord
   belongs_to :need
   belongs_to :user
 
+  enum category: { 'Note': 'note',
+                   'Successful Call': 'phone_success',
+                   'Left Message': 'phone_message',
+                   'Failed Call': 'phone_failure' }
```

Comment:
Heads up that the database will contain needs of category `phone_import`. We'll need to render those with a friendly name like "Imported call log", but you shouldn't be able to add a _new_ note of type `phone_import`. We can leave that to a trello card [like this one](https://trello.com/c/LUGa0GGa/281-improve-rendering-of-imported-note-contents) so it doesn't block merging this PR though.

Reply:
ok - noted.